### ISSUE TITLE
Home: Accessiblity

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-www#103]
From Lighthouse report at https://foo-software-lighthouse-reports.s3.amazonaws.com/report-1619641208795.html#accessibility

#### Acceptance Criteria:

- [x] add a <title> element: 
       `MIT OpenCourseWare | Free Online Course Materials`
- [x] (add a `<title>` to the search page while we're at it)  
- [x] add [alt] attributes for all images
- [ ] Test the results with Lighthouse
- [x] add a label for the search input
- [x] add a label for the newsletter sign up input 
- [x] refactor HTML heading tags so that they are consistently descending (can't have an h5, then an h4, then an h5)

#### Related issues

- Many of the same items appear in the SEO issue #9. 
- #105 Update Give now and Learn More button text color

**Comment by noisecapella** [comment]
The missing title is likely a build configuration issue. We have it set in local development

**Comment by noisecapella** [comment]
I think that should be everything in this issue
